client/js/bundles/procedure/administrationListBoilerplate.js

```diff
@@ -19,4 +19,16 @@ const components = { DpFlyout, DpButton }
 
 initialize(components).then(() => {
   AnimateById()
+
+  document.addEventListener('DOMContentLoaded', () => {
+    const deleteButton = document.querySelector('[data-cy="deleteSelectedBoilerplate"]')
+    const checkboxes = Array.from(document.querySelectorAll('input[data-checkable-item]'))
+
+    deleteButton.addEventListener('click', () => {
+      if (checkboxes.some(checkbox => checkbox.checked) &&
+        confirm(Translator.trans('check.entries.marked.delete'))) {
+        deleteButton.closest('form').submit()
+      }
```

Comment:
What do you think about adding this?

```
if (!checkboxes.some(checkbox => checkbox.checked)) {
  event.preventDefault()
  dplan.notify.error('Bitte markieren Sie mindestens einen Eintrag.')
}
```
(with a trans key of course)

I think it could improve user experience when clicking the button without having items selected.